src/style.css

```diff
@@ -2,0 +1,70 @@
+body {
+  margin-right: 8px;
+  margin-left: 8px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.input {
+  margin-bottom: 20px;
+}
```

Comment:
You are applying bottom margin to the `.input` class. It's a good practice to be consistent with your margins. Try to add only top or bottom, but not both, to avoid potential margin collapse.